### DISCUSSION TITLE
Finding `always` docs in Xstate is impossible using search

### DISCUSSION THREAD

**Discussion by milkbump** [statelyai/xstate#1548] [Q&A]
I always resort to search to quickly find stuff in docs. However, Algolia finds occurrences of the word "always" used in typical English rather than the `always` transient-ish state similar to `on { '': {} }`.

Is there an easy way to solve this? 

My first thought was removing unneeded occurrences of words that conflict with APIs (e.g `always`) to make search easier. But in the name of "tools not rules", you'd need to lint doc files to make sure contributors don't let words slip in.

**Comment by davidkpiano** [comment]
This is a tough problem to solve, because there are other terms that are common in the English language that are also part of the API (such as `on`).

There might be a way to rank certain terms higher; would love some help with this.

**Reply by megleighc** [reply]
Not sure if this would help, but as a short term solution I see in the docs that one of the most helpful lines for "always" here:
> Eventless transitions are defined on the **always** property of the state node

doesn't get picked up and inserted into the search list (assuming because it's formatted as code I think?). 

You could add a similar sentence right under the _Eventless Transitions_ section referencing "always property of the state node" or something, and it might help for those wanting a quick hit to the section related to this.

**Reply by davidkpiano** [reply]
Thanks, that gives me an idea... would this help?

<img width="486" alt="Screen Shot 2020-10-21 at 3 12 46 PM" src="https://user-images.githubusercontent.com/1093738/96771609-eb21d680-13af-11eb-9f13-352c2075214d.png">

**Reply by megleighc** [reply]
I think that's perfect!

**Comment by milkbump** [comment]
In such situations, I typically have an easier time using single page docs such as [rollup docs](https://rollupjs.org/guide/en/), [svelte docs](https://svelte.dev/docs) or [romefrontend.dev](https://romefrontend.dev/). 

As a last resort, you can `ctrl+f` and scroll until you see something highlighted in yellow that looks like what you're looking for. Navigating multiple pages using the sidebar or "next page" buttons is extremely sluggish.
